build.sh

```diff
@@ -1,14 +1,31 @@
-#!sh
+#!/bin/sh
+set -e
 
-cd VirtualBox
+# Script to grab binaries that are going to be bundled with windows installer.
+# Note to maintainers: Update versions used below with newer releases
 
-#http://www.virtualbox.org/manual/ch02.html
-#VirtualBox-4.3.10-93012-Win.exe -extract -silent -path .
-#VirtualBox-4.3.12-93733-Win -extract -silent -path .
-#http://www.catonrug.net/2013/03/virtualbox-silent-install-store-oracle-certificate.html
+(
+	mkdir -p "VirtualBox" && cd "VirtualBox"
 
-cd ../Boot2Docker
+	# http://www.virtualbox.org/manual/ch02.html
+	curl -s -L -o virtualbox.exe http://download.virtualbox.org/virtualbox/4.3.26/VirtualBox-4.3.26-98988-Win.exe
+	virtualbox.exe -extract -silent -path . && rm virtualbox.exe # not neeeded after extraction
+	rm *x86.msi # not needed as we only install 64-bit
+	# http://www.catonrug.net/2013/03/virtualbox-silent-install-store-oracle-certificate.html
+)
+(
+	mkdir -p "Boot2Docker" && cd "Boot2Docker"
 
-curl -L -o boot2docker.iso https://github.com/boot2docker/boot2docker/releases/download/v0.9.1/boot2docker.iso
+	curl -s -L -o boot2docker.iso https://github.com/boot2docker/boot2docker/releases/download/v0.9.1/boot2docker.iso
+	curl -s -L -o boot2docker.exe https://github.com/boot2docker/boot2docker-cli/releases/download/v0.9.2/boot2docker-v0.9.2-windows-amd64.exe
+)
+(
+	mkdir -p "msysGit" && cd "mSysGit"
```

Comment:
I know the actual FS is case insensitive, but can we at least be self-consistent and have both of these be `msysGit`? :+1: